src/main/flight/pid_luxfloat.c

```diff
@@ -80,22 +84,24 @@ STATIC_UNIT_TESTED int16_t pidLuxFloatCore(int axis, const pidProfile_t *pidProf
 
     SET_PID_LUX_FLOAT_CORE_LOCALS(axis);
 
+    pidInitFilters(pidProfile);
```

Comment:
This should indeed be set outside PID itself. Lets say some legacy stayed in here after many changes ;p